style.css

```diff
@@ -61,6 +61,40 @@ nav a:hover{
     margin: 20px;
     background-color: whitesmoke;
 }
+/********* WAIT LIST ***********/
+.container {
+    width: 30%;
+    display: grid;
+    border: 3px solid gray;
+    border-radius: 15px;
+    margin: 50px auto;
+    height: 480px;
+    background-color: white;
+    padding: 15px;
+}
+.test {
```

Comment:
can you give the class a more descriptive name? `test` seems like something you were trying out and not the final one.

Reply:
i was just using random class names that entered my head. i will work on it now